src/components/Navbar/SettingsDrawer/WordByWordSection.tsx

```diff
@@ -217,15 +213,31 @@ const WordByWordSection = () => {
       </Section.Row>
       <Separator className={styles.separator} />
       <Section.Row>
-        <Section.Label>{t('trans-lang')}</Section.Label>
-        <Select
-          size={SelectSize.Small}
-          id="wordByWord"
-          name="wordByWord"
-          options={WORD_BY_WORD_LOCALES_OPTIONS}
-          value={wordByWordLocale}
-          disabled={shouldDisableLanguageSelect}
-          onChange={onWordByWordLocaleChange}
+        <DataFetcher
```

Comment:
I found 2 issues:

1. There is an issue with the loading indicator:
<img width="441" alt="Screenshot 2024-07-04 at 7 42 58 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/d07b4939-2bda-45e4-8515-cbf0382311a5">

2. I think `<Section.Label>{t('trans-lang')}</Section.Label>` should still be there while it's loading or error since now if it's in the loading or error states, user does not know which section label this is. 
<img width="435" alt="Screenshot 2024-07-04 at 7 45 10 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/e08d553d-10c5-4dba-bd89-646773fccc6d">